cfg/1.8/config.yaml

```diff
@@ -9,36 +9,13 @@
 
 master:
   apiserver:
-    confs:
-      - /etc/kubernetes/manifests/kube-apiserver.yaml
-      - /etc/kubernetes/manifests/kube-apiserver.manifest
```

Comment:
I think we still need the possibility of .manifest extension (as an alternative to .yaml) for kops and kubespray installations (see https://github.com/aquasecurity/kube-bench/commit/c44e0db97b5676c980de3dd87ba52b1a77eed478)

Reply:
(same for scheduler, controller manager & etcd below in this file)

Reply:
I don't see anything wrong with keeping them. My idea was to clean up our configs a bit and there is no actual problem keeping them.